api/osf_api.py

```diff
@@ -197,17 +183,46 @@ def get_existing_file(session, node_id=settings.PREFERRED_NODE):
         return upload_fake_file(session, node)
 
 
-def upload_fake_file(session, node=None, name='osf selenium test file for testing because its fake.txt', upload_url=None, provider='osfstorage'):
+def upload_single_quickfile(session):
+    """Upload a new quickfile. Delete existing quickfiles first.
+    Note: Currently using v2.0 of the API. Certain lines will need to be changed on update.
+    TODO: Make this more general.
+    """
+    user = current_user(session)
+    quickfiles_url = user.relationships.quickfiles['links']['related']['href']
+    delete_all_quickfiles(session, quickfiles_url)
+
+    upload_url = user.relationships.quickfiles['links']['upload']['href']
+    return upload_fake_file(session, upload_url=upload_url, quickfile=True)
+
+
+def delete_all_quickfiles(session, quickfiles_url):
+    """ Delete all quickfiles. Just pass in the quickfiles url for the currently logged in user.
+    """
+    for quickfile in session.get(quickfiles_url)['data']:
+        delete_url = quickfile['links']['delete']
+        delete_file(session, delete_url)
```

Comment:
Will multiple browsers affect the same user? If I one browser tries to delete a file that has just been deleted by another browser, `delete_file` may fail.

Reply:
Nope, each browser has their own account across all test environments.

- Selenium Firefox User
- Selenium Chrome User
- Selenium Edge User

We are safe from cross-user deletes here.

Reply:
Ah make sense. I realize that I was thinking about the add-on files thing we did last time where different users still connect to the same addon.